docs/tutorials/motor_noise_debugging.rst

```diff
@@ -194,3 +194,80 @@ vibration resistant features include, but are not limited to the following:
 
 Additionally, consider making redundant ground connections through a fuse or polymeric positive temperature coefficient device (PTC) to prevent high current flows through 
 a carbon fiber frame.
+
+Grounding Your Vertiq Module's Frame
+#####################################################
+
+Vertiq's Pro Kit modules, as well as many modular motors in the 40, 60, and 81 diameter families are shipped with a user accessible, nonpopulated component footprint. 
+This footprint provides a convenient method for connecting the module’s frame to the V- electrical ground through a mounting screw. Illustrations of the 40, 60, 
```

Comment:
a similar thing as above, but could we potentially move these pictures to the module family pages? Just tell people to go look at their family page to find it? Just protects us from ourselves forgetting to update this.